prototype_source/fx_graph_mode_ptq_dynamic.py

```diff
@@ -239,9 +239,13 @@ def evaluate(model_, data_source):
     .set_object_type(nn.LSTM, default_dynamic_qconfig)
     .set_object_type(nn.Linear, default_dynamic_qconfig)
 )
-# Deepcopying the original model because quantization api changes the model inplace and we want
+# Load model to create the original model because quantization api changes the model inplace and we want
 # to keep the original model for future comparison
-model_to_quantize = copy.deepcopy(model)
+
+# Load the model
+model_to_quantize = torch.load(model_data_filepath + 'word_language_model_quantize.pth', map_location=torch.device('cpu'))
```

Comment:
I don't think this will work: `model_to_quantize` will be a dictionary, not a `nn.Module`, and the next line will fail.
Have you verified it?

I think you need to copy the logic starting with the line 164: create `LSTMModel` and then use `load_state_dict`.

Reply:
you are right that makes sense. I will effect the changes and create a PR